cmd/controller-manager/app/options/options.go

```diff
@@ -100,6 +100,7 @@ func (s *ControllerManagerServer) AddFlags(fs *pflag.FlagSet) {
 	fs.Var(k8scomponentconfig.IPVar{Val: &s.Address}, "address", "DEPRECATED: see --bind-address instead")
 	fs.MarkDeprecated("address", "see --bind-address instead")
 	fs.Int32Var(&s.Port, "port", 0, "DEPRECATED: see --secure-port instead")
+	fs.IntVar(&s.ConcurrentSyncs, "coroutines", defaultConcurrentSyncs, "Coroutines number")
```

Comment:
I think `coroutines` may confuse people, since it only applies to concurrent broker synchronizations. How about we call it `concurrent-syncs` so that the flag matches the internal code setting?

Reply:
Thanks for review it. If so, I agree with you and I will recheck it again.